packages/fxa-auth-server/lib/oauth/db/accessToken.js

```diff
@@ -40,7 +40,7 @@ class AccessToken {
     /** @type {Buffer} */
     this.token = token || unique.token();
     /** @type {Date} */
-    this.createdAt = createdAt || new Date();
+    this.createdAt = createdAt || new Date(new Date().setMilliseconds(0));
```

Comment:
Dates in mysql were only down to the second. Some tests assumed this and it's easier and more consistent to leave it as such.